package.json

```diff
@@ -44,6 +44,7 @@
     "core-js": "^2.5.7",
     "reflect-metadata": "^0.1.13",
     "rxjs": "6.5.2",
+    "uuidv4": "^5.0.1",
```

Comment:
I couldn't find any changelogs for uuidv4, so I'm not sure whether the previous versions have breaking changes we should be wary of. It's possible there were none, though I wonder how they got all the way to v6 in that case -- maybe they don't use semantic versioning?

Not sure what would be the best thing to do here. 1) We could leave the peer dependency as it is as most users probably have no issues updating to the latest version, or 2) we could sift through git commits and see how the one function we use is changed (if at all) then determine the correct version range from that. Or, 3) we could contact the author, I guess. 

Personally, I think it's probably fine to do #1, but let me know what you think.

Reply:
I checked out the commits in their repo. I am not sure they really follow semver. The changes between v0.5.0 through v6.0.0 seem to be implementation details. I guess it is a fine line between a minor version change and a breaking major version change in some cases, but overall...the public interface of the library does not appear to have changed from v0.5.0 through v6.0.0. So that could probably be our range.